spec/lib/generators/migration_generator_spec.rb

```diff
@@ -3,7 +3,12 @@
 require 'generators/migration/migration_generator'
 
 describe ManageIQ::Schema::MigrationGenerator do
-  include Rails::Generators::Testing::Behaviour
+  if Rails.version >= "7.1"
+    include Rails::Generators::Testing::Behavior
+  else
+    include Rails::Generators::Testing::Behaviour
+  end
```

Comment:
Really? 😞

Reply:
It was more a "consistency in naming" vs. 🇬🇧 / 🇺🇸  spelling fight. https://www.github.com/rails/rails/pull/45180

I do prefer the fight though.  It's more fun.